service/internal/resources.go

```diff
@@ -227,7 +227,7 @@ var _escData = map[string]*_escFile{
 		name:    "component_header.html",
 		local:   "templates/component_header.html",
 		size:    156,
-		modtime: 0,
+		modtime: 1605733049,
```

Comment:
What are those changes?

Reply:
This happened when testing locally and I assumed it was fine. I'll reset this file